py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -903,7 +904,17 @@ def aten_ops_slice(
     )
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.chunk.default)
+def chunk_validator(node: Node) -> bool:
+    meta_data = node.args[0].meta.get("tensor_meta")
```

Comment:
Also check for the key `val` for the shape first, else check `tensor_meta` and if not return False